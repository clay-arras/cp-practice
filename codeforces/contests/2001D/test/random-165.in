1
21
8 11 1 10 16 16 14 24 15 10 4 21 7 16 13 8 3 3 3 21 14
