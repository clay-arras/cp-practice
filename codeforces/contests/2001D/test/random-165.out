13
11 1 24 15 10 4 21 7 16 13 8 3 14
